app/controllers/catalog_controller.rb

```diff
@@ -48,6 +48,7 @@ class CatalogController < ApplicationController
 
     config.show.sidebar_component = Earthworks::Document::SidebarComponent
     config.show.document_component = Earthworks::DocumentComponent
+    config.show.metadata_component = DocumentMetadataComponent
```

Comment:
This should probably be Earthworks::DocumentMetadataComponent

Reply:
I don't understand why we'd bother with namespaces in an application. That makes sense in a library, but for local stuff it just gets wordy.

Reply:
I'd be down to change that, that makes sense. We have ~12 components with the namespace right now though. Consistency?

Reply:
https://github.com/sul-dlss/earthworks/pull/1249